test/Passes/tile-and-fuse-fill-named-op.mlir

```diff
@@ -1,4 +1,18 @@
 // RUN: tpp-opt %s -tile-consumer-and-fuse-producers | FileCheck %s
+// RUN: tpp-opt %s -tile-consumer-and-fuse-producers="keep-named-op" | FileCheck %s -check-prefix=FUSE-NAMED
+
+// This test checks for the fusion of bias and relu with matmul.
+// The unfused version looks like below sequence.
+    // %{{.+}} = linalg.transpose ins(%{{.+}} : tensor<32x32x32x32xf32>) outs(%{{.+}} : tensor<32x32x32x32xf32>) permutation = [0, 1, 3, 2]
+    // %{{.+}} = scf.forall (%{{.+}}, %{{.+}}) in (8, 32) shared_outs(%{{.+}} = %{{.+}}) -> (tensor<8x32x32x32xf32>) {
+    //     %{{.+}} = linalg.fill ins(%{{.+}} : f32) outs(%{{.+}} : tensor<32x32xf32>) -> tensor<32x32xf32>
+    //     %{{.+}} = linalg.generic
+    //        %{{.+}} = linalg.mulf
+    //        %{{.+}} = linalg.addf
+    //     %{{.+}} = linalg.generic
+    //        %{{.+}} = linalg.addf
+    //     %{{.+}} = linalg.generic 
+    //        %{{.+}} = arith.maximumf
```

Comment:
I'd remove the IR example, IMO it makes things more confusing.
Let the CHECK cases do the talking but the top level comment can still be present next to relevant test case.